### DISCUSSION TITLE
Stand up Thursday April 27th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#71] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 -  Work at my apartment :P 
 
**What are your priorities for the next 24 hours?**:
- work with @CristianCazares
- auth Middleware
- implement some tests (maybe)

**Blockers**:
 - I'm sick 😷

**Shoutouts**:
 -@everyone for their hard work

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Connected server with data analysis service 
 - Helped @CristianCazares run backend on his machine 
 
**What are your priorities for the next 24 hours?**:
 - Finished error handling
 - Deploy new changes to GC 
 
**Blockers**:
 - None
 
**Shoutouts**:
 - @everyone for being so productive

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- login and signup calls to api finally on the client!
- Server error handling implemented. Probably needs refactor.

**What are your priorities for the next 24 hours?**:
- Implement contexts
- Protect routes
- Add backend changes to the most recent branch

**Blockers**:
- none

**Shoutouts**:
- thanks to @BrandonMagana for "pulling it out" on the psql set up.

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
 - Adapted the cleaning script for the excel format it's received 
 
**What are your priorities for the next 24 hours?**:
 - Decide what are we going to compare or plot
 - Error handling  
 
**Blockers**:
 - none
 
**Shoutouts**:
 -@todos
